packages/components/src/components/post-language-option/post-language-option.scss

```diff
@@ -48,18 +48,18 @@ post-language-option {
   }
 
   &:where([variant='menu']) {
-    width: 100%;
+    min-width: 73px;
```

Comment:
I know this is the exact width specified in Figma. But please can we ask the designers if we can use an even number like 80px, because:
1. It feels odd to use such a random number
2. The 73px are based on the trigger width in Figma which is using "DE" as the selected language. But with a different language, the trigger would have a different width for sure.